wurzel/steps/qdrant/step_multi_vector.py

```diff
@@ -49,51 +51,21 @@ def _create_collection(self, size: int):
             replication_factor=self.settings.REPLICATION_FACTOR,
         )
 
-    def run(self, inpt: DataFrame[EmbeddingMultiVectorResult]) -> DataFrame[QdranttMultiVectorResult]:
+    def run(self, inpt: DataFrame[EmbeddingMultiVectorResult]) -> DataFrame[QdrantMultiVectorResult]:
+        log.debug(f"Creating Qdrant collection {self.collection_name}")
         if not self.client.collection_exists(self.collection_name):
             self._create_collection(len(inpt["vectors"].loc[0][0]))
         df_result = self._insert_embeddings(inpt)
         return df_result
 
-    def _insert_embeddings(self, data: DataFrame[EmbeddingMultiVectorResult]) -> DataFrame[QdranttMultiVectorResult]:
-        log.debug(f"Creating Qdrant collection {self.collection_name}")
+    def _insert_embeddings(self, data: DataFrame[EmbeddingMultiVectorResult]) -> DataFrame[QdrantMultiVectorResult]:
+        log.info("Inserting embeddings", extra={"count": len(data), "collection": self.collection_name})
```

Comment:
insert embeddings is not not longer to overload, isnt it(maybe only for typing)?